.github/workflows/ci.yml

```
@@ -70,4 +70,7 @@ jobs:
           fi
 
       - name: Type Check
-        run: yarn tsc --noEmit
\ No newline at end of file
+        run: yarn tsc --noEmit
+        
```

yamllint (1.29.0-1)

[error] trailing spaces

(trailing-spaces)